joomla/joomla-cms
# Create a compressed version of the send test mail JS

#### Summary of Changes

Every core JS file has an uncompressed and compressed version. The sendtestmail.js did not have this. This PR adds the compressed version.
#### Testing Instructions
1. On the Global configuration check if the Send test mail button works
2. Apply the patch
3. Check again if the test mail button still works
4. Turn on debug mode
5. Check again if the test mail button still works
